docs/HowToUseSmartFS.md

```diff
@@ -208,11 +208,7 @@ Below are the optional features configurations. Refer *os/fs/smartfs/Kconfig* fo
 	```
 	File Systems -> SMART file system -> SMARTFS options -> Enable filesystem journaling for smartfs
 	```
```

Comment:
Above description is not applicable for RECOVERY

Reply:
Above description is for CONFIG_SMARTFS_JOURNALING (after point 3)
The description below, after point 4, is for RECOVERY.

Reply:
Same as above. not master branch. we created it as a develop branch, also maintainer already confirmed(Mr. Chang)